packages/core/docs/vibe-3-changelog.md

```diff
@@ -218,6 +218,7 @@ codemod: `avatar-component-migration`
 - `position` changed to only accept "top, right, bottom, left" [codemod]
 - `justify` removed [codemod]
 - `arrowPosition` removed [codemod]
+- `addKeyboardHideShowTriggersByDefault` make default true [codemod]
```

Comment:
what should the codemod do? 
- remove the prop if exists?
- add it as false if it does not exist?